feedparser/http.py

```diff
@@ -71,9 +71,7 @@ class request(object):
 from .datetimes import _parse_date
 from .urls import convert_to_idn
 
-# Python 3.1 deprecated decodestring in favor of decodebytes.
-# This can be removed after Python 2.7 support is dropped.
-_base64decode = getattr(base64, 'decodebytes', base64.decodestring)
+_base64decode = base64.decodebytes
```

Comment:
@tirkarthi Actually, shouldn't all `_base64decode` be replaced by `base64.decodebytes`?
Since no detection is needed anymore and only 1 function is suppored.

Reply:
Safihre, you're correct and I'll address this shortly.